checks.d/redisdb.py

```diff
@@ -325,6 +327,17 @@ def _check_slowlog(self, instance, custom_tags):
 
         self.last_timestamp_seen[ts_key] = max_ts
 
+    def _check_command_stats(self, conn, tags):
+        """Get command-specific statistics from redis' INFO COMMANDSTATS command
+        """
+        command_stats = conn.info("commandstats")
+        for key, stats in command_stats.iteritems():
+            command = key.split('_', 1)[1]
+            command_tags = tags + ['command:%s' % command]
+            self.gauge('redis.command.calls', stats['calls'], tags=command_tags)
+            self.gauge('redis.command.usec', stats['usec'], tags=command_tags)
```

Comment:
I can't think of where I'd use it directly.  Maybe to hunt down where redis is spending it's time on a loaded system?